lms/djangoapps/experiments/flags.py

```diff
@@ -138,6 +138,9 @@ def is_enabled(self, course_key=None):
     def is_enabled_without_course_context(self):
         return self.is_enabled()
 
+    def is_enabled_for_course(self, course_key=None):
+        return super().is_enabled(course_key)
```

Comment:
Why a new function, instead of just calling `is_enabled`?

Reply:
In our situation we need to distinguish between:

- Is the user in control
- Does this experiment apply to this course

Can this be done in a different way?

Reply:
We want to show educators a "preview bar" if the experiment is on for any of their students.